Makefile

```diff
@@ -14,6 +14,8 @@ PATH_BINARY_BUILD_CLI ?= $(GOPATH)/bin
 ARCH_ARM64 ?= arm64
 ARCH_AMD64 ?= amd64
 MAIN = ./cmd/app
+LATEST_RC= $$(git ls-remote --exit-code --sort='v:refname' --tags https://github.com/iancardosozup/horusec.git --ref 'v*.*.*-rc.*' | cut --delimiter='/' --fields=3 | tail --lines=1 | sed 's/.*\///; s/\^{}//')
+LATEST_BETA= $$(git ls-remote --exit-code --sort='v:refname' --tags https://github.com/iancardosozup/horusec.git --ref 'v*.*.*-beta.*' | cut --delimiter='/' --fields=3 | tail --lines=1 | sed 's/.*\///; s/\^{}//')
```

Comment:
Shouldn't be `https://github.com/horusec/horusec.git`?

Reply:
actually is `https://github.com/ZupIT/horusec.git` ty